New version: DuckStudio.FufuDevTools version 2025.02.20.1150

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #231585

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## External requirements

- ⚠️ Should be reviewed by @DuckDuckStudio

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/231713)

Dragon1573 committed:
New version: DuckStudio.FufuDevTools version 2025.02.20.1150

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

DuckDuckStudio commented on manifests/d/DuckStudio/FufuDevTools/2025.02.20.1150/DuckStudio.FufuDevTools.locale.zh-CN.yaml:
```diff
@@ -0,0 +1,44 @@
+- 校对
+- 检查
+- 芙芙工具箱
+ReleaseNotes: |-
+  修复
+  - fix: 一些错误修复 by @DuckDuckStudio @fjwxzde in #39
+  Full Changelog: 2025.02.14.1314...2025.02.20.1150
+ReleaseNotesUrl: https://github.com/DuckDuckStudio/Fufu_Dev_Tools/releases/tag/2025.02.20.1150
```
zh-CN区域清单外的其他区域清单中的 `ReleaseNotes` 不正确，应修改:
1. 在其他区域清单中添加和 zh-CN 区域清单中一样的 `ReleaseNotes` 并添加 `ReleaseNotesUrl`
或
2. 在其他区域清单中添加 `ReleaseNotesUrl`，移除其他区域的 `ReleaseNotes`

Dragon1573 committed:
Apply suggestions from code review

- Remove "ReleaseNotes" from all locales EXCEPT "zh-CN".
- Add "ReleaseNotesUrl" for all locales.

stephengillie commented:
Automatic Validation ended with:
> 

(Automated response - build 980.)

stephengillie commented:
Automatic Validation ended with:
> 

(Automated response - build 980.)
